doc/base/classes.xml

```diff
@@ -5083,10 +5083,10 @@
 </class>
 <class name="Area2D" inherits="CollisionObject2D" category="Core">
 	<brief_description>
-		General purpose area detection and influence for 2D physics.
+	  General purpose area detection for 2D physics. Can override 2D physics properties within range.
 	</brief_description>
 	<description>
-		General purpose area detection for 2D physics. Areas can be used for detection of objects that enter/exit them, as well as overriding space parameters (changing gravity, damping, etc). For this, use any space override different from AREA_SPACE_OVERRIDE_DISABLE and point gravity at the center of mass.
+		General purpose area detection for 2D physics (see [CollisionObject2D] for usage). Areas can detect objects that enter or exit them. They can also override space parameters such as gravity, damping, etc). For this, set the 'space_override' property to any value other than AREA_SPACE_OVERRIDE_DISABLE and point gravity at the center of mass.
```

Comment:
Small suggestion, but also a general writing tip: `Areas can detect` -> `Areas detect`. I would remove the first sentence:
`General purpose area detection for 2D physics (see [CollisionObject2D] for usage). Areas can detect objects that enter or exit them.`
-> Full suggestion:
`2D area that detects nodes that enter or exit it. Change the 'space_override' property to AREA_SPACE_OVERRIDE_* to override physics parameters for nodes like [RigidBody2D]. E.g. gravity, damping... See [CollisionObject2D] for usage.`

Reply:
Done.